src/cond.js

```diff
@@ -28,7 +30,9 @@ var _curry1 = require('./internal/_curry1');
  *      fn(100); //=> 'water boils at 100°C'
  */
 module.exports = _curry1(function cond(pairs) {
-  return function() {
+  var arity = pairs.length ?
+    Math.max.apply(Math, map(function(pair) { return pair[0].length; }, pairs)) : 0;
```

Comment:
We could express this as a fold to obviate the need for the `pairs.length` check.